csharp/src/Google.Protobuf.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.22823.1
+# Visual Studio 2013
+VisualStudioVersion = 12.0.31101.0

```

Comment:
My worries is the mono version we have on Goobuntu. We should double check it works just fine with C# 6 (I don't think it would). I think there's an inherent value in allowing every googler to checkout the code and build it on their machine with the mono version they can apt-get. On the other hand, I can see clear benefits in using C#6 as well, so I am not sure here.